CHANGES.rst

```diff
@@ -4,7 +4,13 @@ CHANGES
 1.2.1 (unreleased)
 ------------------
 
-- Nothing changed yet.
+ - Create migrations for authtools.
+
+   If updating from an older authtools, these migrations must be run on your apps::
+
+     $ python manage.py migrate --fake authtools 0001_initial
+
+     $ python manage.py migrate
```

Comment:
there's a formatting bug, this is coming up as a blockquote see https://github.com/fusionbox/django-authtools/blob/1.8-support/CHANGES.rst